### ISSUE TITLE
A small bug in the latest released version 3.6.0(tar.gz)

### ISSUE THREAD

**Issue by shubihu** [vesoft-inc/nebula-docker-compose#152]
<img width="412" alt="image" src="https://github.com/vesoft-inc/nebula-docker-compose/assets/36021958/f3697c05-10fd-45e9-b613-e0495fdf52dd">

This image should be `docker.io/vesoft/nebula-console:v3.5.0` or `docker.io/vesoft/nebula-console:v3.6.0` , I guess.

**Comment by wey-gu** [comment]
Yes, it should be v3, v3.6, or v3.6.0, may I know where did you encounter this wrong tag? From this repo I quickly checked they are good already.

https://hub.docker.com/r/vesoft/nebula-console/tags

**Comment by wey-gu** [comment]
Oh I see, it's from the tar.gz in release, @Nicole00 could we replace a new tar gz package in the release?
